bin/node/runtime/src/lib.rs

```diff
@@ -609,6 +609,7 @@ parameter_types! {
 	pub const CouncilMotionDuration: BlockNumber = 5 * DAYS;
 	pub const CouncilMaxProposals: u32 = 100;
 	pub const CouncilMaxMembers: u32 = 100;
+	pub const CouncilModuleId: ModuleId = ModuleId(*b"py/counc");
```

Comment:
@gavofyork Can't really avoid this.

`.into_account` takes the raw bytes of the seed and injects it into the address bytes. The instance prefix `I::PREFIX` alone is so many bytes it fills the rest of the address space, which makes it so that two instances generate the same addresses.

Allowing users to specify the `ModuleId` used here for seeding the account I think is the best solution, and what we have done in other places.